docs/website/docs/index.md

```diff
@@ -0,0 +1,13 @@
+---
+id: home
+title: Renovate Docs
+hide_title: true
```

Comment:
I don't know if `hide_title:` is the right strategy here.
Why don't we ensure that `title:` is the proper `h1` text?

Also some `h1` tags take up a lot of screen space, check out the `http://localhost:3000/php` page on your build.